src/Call/Oneclick/InitPaymentRequest.php

```diff
@@ -45,11 +43,11 @@ public function __construct(
 		string $merchantId,
 		string $origPayId,
 		string $orderId,
-		int $totalAmount = null,
-		Currency $currency = null,
+		Cart $cart = null,
```

Comment:
@janlanger Jaký je tvůj názor na tuto změnu? Csob dokumentace popisuje, že `totalAmount` a `currency` musí být buďto obojí null, nebo obojí musí být hodnota. Když místo toho předám celý `Cart`, nemusím řešit, jestli náhodou není jedno z toho nenulové a druhé nulové. Na druhou stranu Cart si s sebou nese i seznam položek a ty se v tomto endpointu nevyužijí. Za mě se mi to s Cart líbí o trochu víc.